### PR TITLE
Document about item action plugin ordering.

### PR DESCRIPTION
**Pull request by draghuram**
Thank you for contributing to Velero!

# Please add a summary of your change

Added clarification that item action plugins have no order.  Please feel free to suggest changes.

# Please indicate you've done the following:

- [Yes ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by draghuram** [comment]
@sseago We discussed this doc change few weeks ago in the call and you suggested a change regarding plugins registered by the same binary. I made the change to include that case. Please take a look.

**Review by sseago — DISMISSED** [review]

File: site/content/docs/main/custom-plugins.md [hunk 1]
```diff
@@ -59,6 +59,11 @@ Velero supports the following kinds of plugins:
 - **Restore Item Action** - executes arbitrary logic for individual items prior to restoring them into a cluster
 - **Delete Item Action** - executes arbitrary logic based on individual items within a backup prior to deleting the backup
 
+Plugin binaries are discovered by recursively reading a directory in no particular order. Hence no guarantee is provided for the
+order in which item action plugins are invoked. How ever, if a single binary implements multiple item action plugins,
```
Comment (shubham-pampattiwar) [inline site/content/docs/main/custom-plugins.md:63]: s/How ever/However

File: site/content/docs/main/custom-plugins.md [hunk 2]
```diff
@@ -59,6 +59,11 @@ Velero supports the following kinds of plugins:
...
+they may be invoked in the order in which they are registered but it is best not depend on this
```
Comment (shubham-pampattiwar) [inline site/content/docs/main/custom-plugins.md:64]: s/best not/best **to** not

File: site/content/docs/main/custom-plugins.md (hunk 1 above)
Comment (draghuram) [reply site/content/docs/main/custom-plugins.md]: Thanks. Made the change.

File: site/content/docs/main/custom-plugins.md (hunk 2 above)
Comment (draghuram) [reply site/content/docs/main/custom-plugins.md]: Fixed it. Thanks.

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by sseago — APPROVED** [review]

**Pull request merged**